src/pages/Homepage.svelte

```diff
@@ -71,6 +85,7 @@
   <title>{title}</title>
 </svelte:head>
 
+<CompatibilityWarningDialog />
```

Comment:
We don't need this here as it's already in App.svelte.

Reply:
Ahh yes. https://github.com/microbit-foundation/ml-trainer/pull/185/commits/e2f17fed4eae392fe03939c0ae1186c9e9af9177